comfyui_embedded_docs/docs/GeminiVideoOmni/zh-TW.md

```
@@ -1,25 +1,23 @@
-# GeminiVideoOmni
+# Google Gemini Omni（影片）
 
-使用 Google Gemini Omni Flash 模型，根據文字提示生成附帶音訊的影片。可選擇性地提供參考圖片和/或影片來引導或編輯結果。請直接在提示中描述所需的長度（3-10 秒）和長寬比（16:9 或 9:16）。
-
-## ## 輸入
+使用 Google Gemini Omni Flash 模型，根據文字提示生成附帶音訊的影片。可選擇性地提供參考圖片和/或影片來引導或編輯生成結果。直接在提示中描述所需的長度（3-10 秒）和長寬比（16:9 或 9:16）。
 
+## 輸入
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)